inkcut/job/models.py

```diff
@@ -180,6 +180,12 @@ class Job(Model):
     plot_weedline_padding = ContainerList(
         Float(), default=[10, 10, 10, 10]).tag(config=True)
 
+
+    #  I don't understand how to declare an empty list of Layers (for the layers' settings)
+    layercbbb = ContainerList( Bool(),default=[True,True,True,True,True,True,True,True,True,True] ).tag(config=True)
```

Comment:
You should just be able to use:
```python
layers = ContainerList(Layer)
```

Saving it in the config doesn't really make sense here as inkcut currently doesn't save recently run jobs (only keeps them in memory).

Reply:
because when launched inkcut reloads the last SVG used, it make senses to me to reload also the layer setup last used.